recipes/jupyterlab-flake8/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jupyterlab-flake8" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_flake8-{{ version }}.tar.gz
+  sha256: 952af03d0a7361325b9829ddbb86265c1bed645406093f5a1b9baf5058e0311c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - jupyter_packaging >=0.10,<1.dev0,<2
```

Comment:
Looking at the upstream `pyproject.toml`, shouldn't this just be `- jupyter_packaging >=0.10,<2`